tests/integration/test_compute_flow.py

```diff
@@ -292,22 +274,6 @@ def run_compute_test(
             time.sleep(5)
 
         print(f"got status: {status}")
-        output = None
-        for i in range(len(status["results"])):
-            result = None
-            result_type = status["results"][i]["type"]
-            print(f"Fetch result index {i}, type: {result_type}")
-            result = ocean_instance.compute.result(
-                dataset_and_userdata.asset, service, job_id, i, consumer_wallet
-            )
```

Comment:
I removed this because it was unused. The result is not asserted in any way. Since the compute flow tests use sample files, there is no real output of the compute job. We are checking result_logs in many places, though.

Reply:
I think instead of removing it, we should perhaps find a way to assert the result.

Reply:
we do, we assert the output itself :D